app/assets/javascripts/profile_table.js

```diff
@@ -0,0 +1,77 @@
+var ProfileTable = (function(){
+  var widget, selector;
+
+  ProfileTable.prototype = {
+    append: function(){
+      widget = $(selector).editableTableWidget();
+      widget.on('change', parseTableToJSON.bind(this));
+
+      addClickListenersToAddRow();
+      addClickListenersToDeleteRow();
+      parseTableToJSON();
+    }
+  };
+
+  function parseTableToJSON(){
+    $("#testing_ground_technology_profile").text(tableToProfile());
+  };
+
+  function tableToProfile(){
+    var technologyProfile = _.map(tableRows(), function(tableRow){
+      return rowToTechnologyObject(tableRow);
+    });
+    return JSON.stringify(_.groupBy(technologyProfile, 'node'));
```

Comment:
Totally agree with the underscore dependency. The less dependencies the better.

The `map` was because `$.map` completly didn't do what I expected, maybe 'plain' map will be a better option. I'll look into this. For now I could just copy the `groupBy` and use the `plain` map, and remove underscore.